usecase/usecase-webapi-xwalk-tests/samples/IAPXiaomiStore/iap/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "IAP Xiaomi Store Demo",
+  "start_url": "iap.html",
+  "xwalk_package_id": "com.sdk.migame.payment",
+  "xwalk_extensions": ["."],
+  "xwalk_android_permissions": [
+    "GET_TASKS",
+    "GET_ACCOUNTS",
+    "com.xiaomi.sdk.permission.PAYMENT"
```

Comment:
Have you verified these permissions with app-tools? Are them the correct written?